README.md

```diff
@@ -196,7 +196,8 @@ Wisper.subscribe(MyListener.new)
 
 In a Rails app you might want to add your global listeners in an initalizer.
 
-Global listeners are threadsafe.
+Global listeners are threadsafe. Subscribers will receive events from publishers on current and future threads owned by the ruby process.
+
```

Comment:
Would it be accurate for this read "Subscribers will receive events published on all threads"?

Reply:
@krisleech I like it and have applied the wording to both readme changes ✅